tests/testsuite/bench.rs

```diff
@@ -267,9 +267,9 @@ fn many_similar_names() {
         .build();
 
     p.cargo("bench")
-        .with_stdout_contains("test bin_bench ... bench:           0 ns/iter (+/- 0)")
-        .with_stdout_contains("test lib_bench ... bench:           0 ns/iter (+/- 0)")
-        .with_stdout_contains("test bench_bench ... bench:           0 ns/iter (+/- 0)")
+        .with_stdout_contains("test bin_bench ... bench:           0.00 ns/iter (+/- 0.00)")
```

Comment:
in case it changes again in the future, could we use `[..]` to omit everything after the colon?